crates/holochain/src/conductor/conductor.rs

```diff
@@ -1491,6 +1515,18 @@ mod builder {
                 dev_settings: parking_lot::RwLock::new(DevSettings::default()),
             });
 
+            {
+                let handle = handle.clone();
+                tokio::task::spawn(async move {
+                    while !shutting_down.load(std::sync::atomic::Ordering::Relaxed) {
+                        tokio::time::sleep(std::time::Duration::from_secs(60)).await;
+                        if let Err(e) = handle.prune_p2p_agents_db().await {
```

Comment:
I wonder if this will make our tests take up to 60 secs to finish?

Reply:
no, the tests can shutdown with outstanding sleeps like this